src/lib/tenant/configs/contracts/scroll.ts

```diff
@@ -1,5 +1,6 @@
 import {
   AgoraGovernor__factory,
+  AgoraToken__factory,
```

Comment:
is `AgoraToken` the same as `L2GovToken`?

Reply:
yep, happy to rename it to L2GovToken if it makes more sense.